specification/cognitiveservices/HealthInsights/healthinsights.oncophenotype/tspconfig.yaml

```diff
@@ -1,10 +1,3 @@
-emit: [
-  "@azure-tools/typespec-csharp",
-  # "@azure-tools/cadl-python",
-  # "@azure-tools/typespec-java",
-  # "@azure-tools/cadl-typescript",
-]
-
 options:
```

Comment:
I think we want autorest otherwise it will not output the swaggers https://github.com/Azure/azure-rest-api-specs/blob/main/specification/contosowidgetmanager/Contoso.WidgetManager/tspconfig.yaml#L13

Reply:
I was told these folders are only for generating clients and should definitely **not** output autorest

Reply:
In general I'm not aware of any cases where we only have sdk client generation and no autorest. The autorest output is needed for other things like linting as well as documentation publishing and so without it those things will not happen.

Reply:
@markcowl: Please consider this during your design review of this spec and it's folder structure.